lib/modules/platform/gitea/index.ts

```diff
@@ -1,4 +1,5 @@
 import is from '@sindresorhus/is';
+import pMap from 'p-map';
```

Comment:
use this instead
https://github.com/renovatebot/renovate/blob/9bf06584aa9b2398a8a7914150fdf711e2a6f3b4/lib/util/promises.ts#L54

Reply:
I looked at that code but doesn't it expect callback function to return single element not array? If I read code right...

Reply:
wrong, `Iterable` can be an array.

Reply:
oh, ok I got it now, it's just wrapper with default settings. @viceice fixed